### PR TITLE
Cleanup according to PHPStorm inspection part 1: Update PHPDoc

### PR DESCRIPTION
**Pull request by MrPetovan**
Redux of #6401
Part of #4176
Related to #6406, #6407, #6408

This is an impact-less PR, only adding the `@throws` PHPDoc tags where required. We started using Exception but there are few places where we actually catch them. As such most function are susceptible to throw one or more different exceptions, and it should be reflected in the PHPDoc comment.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -42,7 +51,7 @@
 		"npm-asset/cropperjs": "1.2.2",
 		"npm-asset/imagesloaded": "4.1.4",
 		"monolog/monolog": "^1.24"
-	},
+    },
```
Comment (JeroenED) [inline composer.json]: Shouldn't this be a tab instead of 4 spaces?

Comment (MrPetovan) [reply composer.json]: Yup, good catch.

**Comment by MrPetovan** [comment]
Conflict fixed.

**Comment by annando** [comment]
Conflicts again 😁

**Comment by MrPetovan** [comment]
Solved.

**Pull request merged**